.github/workflows/nightly_release.yml

```diff
@@ -12,7 +12,7 @@ permissions:
 jobs:
   build_test_upload:
     if: |
-      github.repository == 'pytorch/data' && github.ref_name == 'main'
+      (github.repository == 'pytorch/data' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch'
```

Comment:
Adding this flag to allow manual workflow_dispatches to run on feature branches, to make testing easier. Going to leave this in as it seems useful in general